### PR TITLE
adding readme.md

### PR DESCRIPTION
**Pull request by ngminhtrung**
@khanh19934 : Please review and accept to merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello world!
```
Comment (ghost) [inline README.md:1]: ## start review

Comment (MedianNguyen) [reply README.md]: finish review

**Comment by ngminhtrung** [comment]
LGTM

**Comment by ngminhtrung** [comment]
LGTM @MedianNguyen 
@tamtm

**Pull request merged**